apps/code-examples/src/app/code-examples/ag-grid/data-entry-grid/inline-help/inline-help.component.html

```diff
@@ -1 +1,5 @@
-<sky-help-inline class="sky-control-help" (actionClick)="onHelpClick()" />
+<sky-help-inline
+  class="sky-control-help"
+  ariaLabel="More information about {{ displayName }}"
```

Comment:
```suggestion
  ariaLabel="Information about {{ displayName }}"
```
We don't really need "More" in any of these labels. I'm going to add suggestions to remove it throughout.